src/hooks/__data__/mock-data.ts

```diff
@@ -588,3 +589,266 @@ export const mockedSecrets: SecretKind[] = [
     },
   },
 ];
+
+// Banner Related
+function formatTimeUTC(date: Date): string {
+  const h = date.getUTCHours().toString().padStart(2, '0');
+  const m = date.getUTCMinutes().toString().padStart(2, '0');
+  return `${h}:${m}`;
+}
+
+const now = new Date();
+const oneHourLater = new Date(now.getTime() + 60 * 60 * 1000);
+const oneHourEarlier = new Date(now.getTime() - 60 * 60 * 1000);
+const halfHourEarlier = new Date(now.getTime() - 30 * 60 * 1000);
+
+const futureStartTime = formatTimeUTC(oneHourLater);
+const pastStartTime = formatTimeUTC(oneHourEarlier);
+const pastEndTime = formatTimeUTC(halfHourEarlier);
+
+const dayOfWeek = now.getUTCDay();
+const dayOfMonth = now.getUTCDate();
+const year = now.getUTCFullYear().toString();
+const month = (now.getUTCMonth() + 1).toString();
+
+// === YAML Banner Generator ===
+function generateBannerYAML({
+  type = 'info',
+  summary = 'Default banner',
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  dayOfWeek,
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  dayOfMonth,
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  year,
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  month,
```

Comment:
Disabling eslint rules is a bad practice and shadow variables can have bad implications on the working of the code inside the function.
Also, Always add a comment why the eslint rule is disabled.

Reply:
Updated.